src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -122,6 +124,7 @@ const ExpandedNode: FC<ExpandedNodeProps> = memo(
           }}
           isAccepted={isAccepted}
           onAdvance={onAdvance}
+          patientRecords={patientRecords}
```

Comment:
Instead of passing this down can we use the PatientRecord context. Remove this line and 374 and then add the following at the beginning of the `ExpandedNodeMemo`
```
const { patientRecords } = usePatientRecords();
```